src/main/java/Duke.java

```diff
@@ -1,10 +1,268 @@
+import java.util.ArrayList;
```

Comment:
Hi, should there be a package statement above this line?
Reference: "Every class should be part of some package."

Reply:
so how does your package look like?

Reply:
it actually depends on how the directory in your project looks like. one of my package statements is "package command"